src/content/docs/errors-inbox/getting-started.mdx

```diff
@@ -84,3 +84,7 @@ You can group APM, Browser, Mobile, Serverless, and Otel entities together and s
 />
 
 <figcaption>**[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Errors inbox, check out your new Workload** </figcaption>
+
+### Entity limits for workloads in errors inbox [#entity-limits] 
+
+Please note: we have a limit of 2,000 entities per workload. 
```

Comment:
I think we should make it more clear that the limit just prevents the use of the workload with errors inbox.  Maybe:

"Please note:  the errors inbox can only be used with workloads containing 2,000 or fewer entities."